### PR TITLE
Replaced pkg/errors dependency

### PR DESCRIPTION
**Pull request by Debanitrkl**
#### What type of PR is this?

/kind cleanup
#### What this PR does / why we need it
Using the native error wrapping feature to omit the direct github.com/pkg/errors dependency.
for https://github.com/kubernetes/release/tree/master/cmd/ci-reporter
#### Which issue(s) this PR fixes:

Refers to: #2549 
#### Does this PR introduce a user-facing change?

```Release Notes
None
```

### CODE REVIEW

**Comment by leonardpahlke** [comment]
Hey @Debanitrkl thanks for opening the PR! :) the code does not compile atm (typo `root.go`) & the release notes comment is not formatted correct. 

Btw, you can check your modifications with `make verify` locally before creating the PR to insure that the test passes :). If you need support feel free to reach out any time :)!

**Review by leonardpahlke — CHANGES_REQUESTED** [review]

**Comment by saschagrunert** [comment]
CI is still not happy :)

File: cmd/ci-reporter/cmd/github.go [hunk 1]
```diff
@@ -19,9 +19,10 @@ package cmd
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"os"
-
-	"github.com/pkg/errors"
+
```
Comment (leonardpahlke) [inline cmd/ci-reporter/cmd/github.go:24]: You need to remove this page break “\n”

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**